pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderToken.java

```diff
@@ -35,7 +35,7 @@
 public class AuthenticationProviderToken implements AuthenticationProvider {
 
     final static String HTTP_HEADER_NAME = "Authorization";
-    final static String HTTP_HEADER_VALUE_PREFIX = "Bearer ";
+    public final static String HTTP_HEADER_VALUE_PREFIX = "Bearer ";
```

Comment:
Rather than making this public, you should make getToken(AuthenticationDataSource) static and call it from the other locations.